docs/markdown/ios/card/subcomponents/card.header.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Card.Header
+description: Card.Header sits at the top, inside of the Card container. It is a required subcomponent that includes pre-text, a title, and navigational elements. The Card.Header should always, at the very least, include a title.
+fullwidth: true
+---
+
+<ImgContainer src="https://i.pinimg.com/originals/11/90/6b/11906b1575ba50e08d48b8a0bf20e805.png" alt="Example of a Card.Header with a highlighted area to show how it is placed and arranged in a Card." noPadding color="background-elevation-accent"/>
+
+
+## Usage guidelines
+
+<TwoCol>
+<Group>
+<Do title="When to use"/>
```

Comment:
These titles don't make sense vis-a-vis the content. Typically the content is answering the title's question, but these seem completely disconnected.